client/test/workflow.test.js

```diff
@@ -35,17 +35,8 @@ describe('Workflow', () => {
       new Scenario(mochaTest)
         .withDomain('ci-test')
         .withDomainAuthorization('ci-test', true)
-        .withFeatureFlags([
-          {
-            key: 'domainAuthorization',
-            value: false,
-          },
-          {
-            key: 'workflowTerminate',
-            value: true,
-          },
-        ])
```

Comment:
are these the default values?  domainAuthorization being false by default makes sense for tests, but I'm not sure what workflowTerminate implies, or if it's relevant to these tests.

Reply:
> are these the default values? domainAuthorization being false by default makes sense for tests, but I'm not sure what workflowTerminate implies, or if it's relevant to these tests.

I updated `withFeatureFlag` fixture in this PR #358 . I just forgot to update this code in that PR. As for what those feature flags mean:

`domainAuthorization`: when enabled, landing on workflow page will make a request to see if user is allowed to view this domain. When disabled, it assumes all users are allowed to view domains.

`workflowTerminate` displays Terminate button in UI when enabled. Will hide it if disabled.

Reply:
rebase or merge would probably remove it then?

but yeah, makes sense and 👍